cmd/loadTests.go

```diff
@@ -888,15 +909,27 @@ func userJourneyThread(frameworkMap *sync.Map, threadWaitGroup *sync.WaitGroup,
 			if err != nil {
 				logError(3, fmt.Sprintf("Unable to create the Application %s: %v", ApplicationName, err))
 				FailedApplicationCreationsPerThread[threadIndex] += 1
+				MetricsWrapper(MetricsController, metricsConstants.CollectorApplications, metricsConstants.MetricTypeCounter, metricsConstants.MetricFailedApplicationCreationCounter)
 				increaseBar(applicationsBar, applicationsBarMutex)
 				continue
 			}
 
 			ApplicationCreationTimeSumPerThread[threadIndex] += applicationCreationTime
+			MetricsWrapper(MetricsController, metricsConstants.CollectorApplications, metricsConstants.MetricTypeGuage, metricsConstants.MetricApplicationCreationTimeGauge, float64(applicationCreationTime))
 			if applicationCreationTime > ApplicationCreationTimeMaxPerThread[threadIndex] {
 				ApplicationCreationTimeMaxPerThread[threadIndex] = applicationCreationTime
 			}
+
+			for _, Status := range app.Status.Conditions{
+				if Status.Type == "Created"{
+					TempTime := Status.LastTransitionTime.Sub(startTimeForApplication)
+					MetricsWrapper(MetricsController, metricsConstants.CollectorApplications, metricsConstants.MetricTypeGuage, metricsConstants.MetricActualApplicationCreationTimeGauge, float64(TempTime))
```

Comment:
```suggestion
					MetricsWrapper(MetricsController, metricsConstants.CollectorApplications, metricsConstants.MetricTypeGuage, metricsConstants.MetricActualApplicationCreationTimeGauge, TempTime.Seconds())
```
Instead of nanoseconds, please push seconds.